joomla/joomla-cms
# [4.0] Fix autoprefixer

Pull Request for Issue https://github.com/joomla/joomla-cms/issues/19219

### Summary of Changes

This changes the scope of the comment that disables the autoprefixer so that it doesn't affect the rest of the file

### Testing Instructions

1. In the front-end, go to Module Settings
2. Ensure the select dropdown doesn't have multiple arrows
